### PR TITLE
151 ci build fails

### PR DESCRIPTION
**Pull request by neopostmodern**

### CODE REVIEW

**Comment by neopostmodern** [comment]
Probably it would be good to merge #157 first, then we can see if this actually passes the CI build.

File: app/components/Sidebar.js [hunk 1]
```diff
@@ -2,10 +2,12 @@
 import React, { Component } from 'react';
 import styles from './Sidebar.scss';
 import type { ApplicationData, BaseCacheEntry } from '../reducers/cache';
-import { homepage, version, bugs } from '../../package.json';
+import packageJson from '../../package.json';
```
Comment (geohiehoe) [inline app/components/Sidebar.js:5]: When do you have to remove the implicit destruction and when not? Why here in line 7 you don't do it, but for example for the SENDGRID_API_KEY you have to do it?

**Review by geohiehoe — APPROVED** [review]
Looks good. I don't fully get until where (and why) we have to move to typescript, and why later we can stick to the current dialect.

**Comment by geohiehoe** [comment]
I agree that we should merge the updated artifacts first.

Why do I see the "rebase and merge" button enabled, although there are unresolved threads?

**Comment by neopostmodern** [comment]
Because I hadn't ticked the right box in settings yet :)

File: app/components/Sidebar.js (hunk 1 above)
Comment (neopostmodern) [reply app/components/Sidebar.js:5]: It's complicated; depends on explicit vs implicit exports – in this case we're importing from JSON (which is experimental anyway) and that has a *default* export consisting of an object, which can be destructured, but not explicit property exports...

**Pull request merged**